hera_cal/types.py

```diff
@@ -0,0 +1,5 @@
+from typing import Literal
+
+Antpair = tuple[int, int]
+Pol = Literal['xx', 'yy', 'ee', 'nn', 'xy', 'yx', 'en', 'ne']
+Baseline = tuple[int, int, Pol]
```

Comment:
This feels like a thing that could go in `utils.py` unless you feel like this is a more intuitive or "standard" place to find these.